MeshLib/Elements/QuadRule4.cpp

```diff
@@ -12,12 +12,20 @@
 
 #include <logog/include/logog.hpp>
 
+#include "Line.h"
 #include "MathLib/GeometricBasics.h"
-
 #include "MeshLib/Node.h"
 
 namespace MeshLib {
 
+const unsigned QuadRule4::face_nodes[4][2] =
```

Comment:
This is almost repeated from the edge_nodes. Quad's don't have faces...

Reply:
Changed implementation.